reactions/src/HALO.cpp

```diff
@@ -110,16 +106,16 @@ static real sigma8d_integrand(const PowerSpectrum& P, real R, real k) {
 
 // Used when input power spectra is given at z and is already modified
 static real sigma_integrand_mgcamb(const PowerSpectrum& P, real R, real k) {
-    return k*k/(2.*M_PI*M_PI) * P(k) * pow2(W(k*R)*Dl_spt/linear_growth(k));
+    return k*k/(2.*M_PI*M_PI) * P(k) * pow2(W(k*R)*Dl_spt/linear_growth(k)/dnorm_spt);
 }
 
 // d(sigma_8^2)/dR
 static real sigma8d_integrand_mgcamb(const PowerSpectrum& P, real R, real k) {
-    return k*k/(2.*M_PI*M_PI) * P(k) * pow2(Dl_spt/linear_growth(k)) * 2.*W(k*R)*Wd(k,R) ;
+    return k*k/(2.*M_PI*M_PI) * P(k) * pow2(Dl_spt/linear_growth(k)/dnorm_spt) * 2.*W(k*R)*Wd(k,R) ;
 }
 
 
-void HALO::scol_init(double vars[]) const{
+void HALO::scol_init(double vars[], bool mgcamb) const{
```

Comment:
Added mgcamb flag to declare what input spectrum is being specified at top level